src/components/Articles/Feed.jsx

```diff
@@ -6,20 +6,40 @@ import Spinner from '../Spinner/Spinner.jsx';
 import { getFeed } from '../../redux/actions/actionCreators';
 import Navbar from '../Layout/Navbar.jsx';
 import '../../assets/css/homepage.css';
+import '../../assets/css/pagination.css';
 
 export class Feed extends Component {
   componentDidMount() {
     this.props.getFeed();
   }
 
+  componentDidUpdate() {
+    if (this.props.articles[13] !== undefined) {
+      const values = [1, 2, 3, 4, 5, 6];
+      if (document.getElementById(`btn${this.props.articles[13].page}`)) {
+        values.forEach((element) => {
+          if (element !== this.props.articles[13].page) {
+            document.getElementById(`btn${element}`).style.backgroundColor = '';
+          }
+          document.getElementById(`btn${this.props.articles[13].page}`).style.backgroundColor = '#367fa9';
+          document.getElementById(`btn${this.props.articles[13].page}`).style.color = '#ffffff';
+        });
+      }
+    }
+  }
+
+  changePage = (e) => {
+    this.props.getFeed(e.target.value);
+  }
+
   render() {
     if (this.props.loading || this.props.articles[0] === undefined) {
       return <Spinner />;
     }
     const { articles } = this.props;
     return (
       <div data-test="homePage" className="feedHome">
-        <Navbar token={articles[14].token} username={articles[14].username} avatar={articles[14].image} />
+        <Navbar token={localStorage.getItem('jwtToken')} username={localStorage.getItem('username')} avatar={localStorage.getItem('image')} />
```

Comment:
if the token is not there that means line `42` will break the application. Kindly fix this.

Reply:
Actually test it before you make a conclusion to this assumption